src/ozone/ozone.py

```diff
@@ -520,6 +521,75 @@ def get_specific_parameter(
 
         return result
 
+    def get_search_results_historical(self, city: str):
```

Comment:
The name for this feature doesn't explain what it does very well. I understand that you're trying to convey that it returns the search results, but for someone who's never used the WAQI historical data portal directly, this might be a little confusing. 

Let's rename this to something along the lines of `get_station_options`. That way this method can also be used for regular users, who aren't interested in historical data.

Reply:
I agree entirely, the method name needs to be replaced. It also needs a docstring, too, for completeness.

I want to rename it but I also want it to become consistent with `get_city_air` (`get`, for the given `city`, its `air` quality) so I propose `get_city_station_options` (`get`, for the given `city`, its possible `station options`).

What do you think? @Milind220 

---

(As a side note, should I rename the new method into `get_city_historical` in the same vein? Or does the current name needs no modification?)